sqlmesh/core/config/scheduler.py

```diff
@@ -296,7 +306,14 @@ def create_plan_evaluator(self, context: Context) -> PlanEvaluator:
         )
 
     def get_default_catalog(self, context: Context) -> t.Optional[str]:
-        return None
+        default_catalog = self.get_client(context.console).get_variable(
```

Comment:
I wonder whether we can DRY up this block a little

Reply:
For context: `MWAAClient` and `AirflowClient` share a common base class with `get_variable` available in it. So we can easily write a function for this.